doc/building.md

```diff
@@ -1285,10 +1285,10 @@ at least the following targets are known to work:
 | ppc64le-linux-gnu        |
 | s390x-linux-gnu          |
 
-`BASE_OS` must be one of "OEL6" for Oracle Enterprise Linux 6 or "Fedora" (if
-not specified "OEL6" will be the default). If the base OS is "Fedora" the
+`BASE_OS` must be one of `OL` for Oracle Enterprise Linux or `Fedora` (if
+not specified `OL` will be the default). If the base OS is `Fedora` the
```

Comment:
Ever so slightly unrelated to your change, but perhaps worth fixing at the same time, even if it means widening the scope of the issue a bit:

There is no default, so suggest dropping the parenthesis part.